v2/pkg/engine/resolve/resolve.go

```diff
@@ -74,6 +74,10 @@ type Resolver struct {
 	subscriptionTools *sync.Pool
 }
 
+func (r *Resolver) Options() ResolvableOptions {
+	return r.options.ResolvableOptions
```

Comment:
Sure, that we need to make it public? `resolve.ResolverOptions` is created in the Router.

Reply:
I just saw your cosmo PR. If you need access to the Apollo flags you can simply pass `apolloCompatibilityFlags` to the preHandler.

Reply:
There is no reason for it to be public besides needing it in the prehandler. But it's just Apollo Compatibility flags, so I am not sure it matters that it is exposed. How would you pass the options to the prehandler? It would have to be done upon creation of the options.